lib/greetLastParam.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var greetings = require('./simpleGreeting.js');
+var hello3 = exports = module.exports = {};
+
+
+hello3.findingResult = function() {
+	var result = greetings.greet(process.argv[process.argv.length-1]);
```

Comment:
this line doesn't do what you expect if they forget to add a command line argument
```
$ node lib/greetLastParam.js
hello /Users/mel/work/js-da/grading-sandbox/simple-test/lib/greetLastParam.js
```